aws_lambda_powertools/utilities/idempotency/__init__.py

```diff
@@ -8,7 +8,17 @@
 from aws_lambda_powertools.utilities.idempotency.persistence.dynamodb import (
     DynamoDBPersistenceLayer,
 )
+from aws_lambda_powertools.utilities.idempotency.serialization.custom_dict import CustomDictSerializer
+from aws_lambda_powertools.utilities.idempotency.serialization.pydantic import PydanticSerializer
```

Comment:
Unfortunately we can't export this globally: we should be able to use the idempotency utility without having Pydantic installed.

Suggestion: don't export globally here, and just keep it inside the serialization folder. This way, when people want to use it, they have to:

`import PydanticSerializer from aws_lambda_powertools.utilities.serialization`

Reply:
Hey, thanks for noticing!
removed on __init__.py and updated examples